frontend/src/pages/Demo/DemoTailwind.tsx

```diff
@@ -41,11 +41,9 @@ const DemoTailwind = () => {
         <Typography.Title3 className="mb-2 text-blue-dark dark:text-white">
           Disabled
         </Typography.Title3>
-        <div className="flex flex-wrap items-center gap-4 mb-4">
-          <Button state="disabled">
-            Medium Disabled
-          </Button>
-          <Button size="large" state="disabled">
+        <div className="mb-4 flex flex-wrap items-center gap-4">
+          <Button disabled={true}>Medium Disabled</Button>
```

Comment:
```suggestion
          <Button disabled>Medium Disabled</Button>
```

We dont need the `={true}`, we can just pass `disabled` by itself.